examples/lzip_reader.rs

```diff
@@ -0,0 +1,24 @@
+use std::{
+    env,
+    fs::File,
+    io::{self, BufReader},
+    time::Instant,
+};
+
+use lzma_rust2::LzipReader;
+
+fn main() -> io::Result<()> {
+    let mut args = env::args();
+
+    let input = BufReader::new(File::open(args.nth(1).unwrap())?);
+    let mut output = File::create(args.next().unwrap())?;
+    let input_len = input.get_ref().metadata()?.len();
+    let start = Instant::now();
+    let mut reader = LzipReader::new(input)?;
+    io::copy(&mut reader, &mut output)?;
+
+    println!("{input_len} in");
```

Comment:
Sometimes you store the input size in the input_len variable and sometimes you don't Please make this consistent over all files (maybe just calculate the input and output lengehts always beforehand in a variable).